packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -94,6 +94,10 @@ export const BaseButtonStyles = css`
         display: flex;
     }
 
+    a.icon-only {
+        padding-top: 6px;
+    }
```

Comment:
I was able to make an adjustment on this in Fluent UI to center consistently across browsers. Here's what I used:

```css
    .control.icon-only {
        padding: 0;
        line-height: 0;
    }
```

Reply:
Cool, I'll add it.